src/SwiftPackage.ts

```diff
@@ -41,6 +41,7 @@ export interface Target {
 // Swift Package Manager dependency
 export interface Dependency {
     identity: string
+    requirement?: object
     url?: string
     path?: string
```

Comment:
Might as well remove `path` given it is never used. I had just assumed the package description would include a package path item given that is the label used in the Package.swift. Looks like I was wrong.

Reply:
Done